app/org/sagebionetworks/bridge/models/schedules/SchedulerFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import org.sagebionetworks.bridge.validators.ScheduleValidator;
+import org.sagebionetworks.bridge.validators.Validate;
+
+public class SchedulerFactory {
```

Comment:
That's not a discussion for a code review... none of the code exists yet, please see the design document I sent out over the last week for an overview of the tasks API.